newsletter/2025-04-01-announcing-a-lesser-loong-a-day.md

```diff
@@ -0,0 +1,35 @@
+---
+slug: announcing-a-lesser-loong-a-day
+title: 周报形式焕新升级：《每日一小龙》！
+authors: [xen0n]
+date: 2025-04-01T14:30:00+08:00
```

Comment:
Maybe bumping it or people might consider this as an Apr. 1st joke.

Reply:
I have drafted a change clarifying this stance in an upcoming revert PR.

Reply:
#352